showtimes_events/src/models/timestamp.rs

```diff
@@ -0,0 +1,26 @@
+use jiff::Timestamp;
+
+use serde::{de::Error as _, ser::Error as _};
+use serde::{
+    de::{Deserialize, Deserializer},
+    ser::{Serialize, Serializer},
+};
+
+pub fn serialize<S>(dt: &Timestamp, serializer: S) -> Result<S::Ok, S::Error>
+where
+    S: Serializer,
+{
+    let ts = dt.as_second();
+
+    u32::try_from(ts)
+        .map_err(|_| S::Error::custom(format!("{dt} cannot be represented as DateTime")))?
+        .serialize(serializer)
+}
+
+pub fn deserialize<'de, D>(deserializer: D) -> Result<Timestamp, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let ts: u32 = Deserialize::deserialize(deserializer)?;
+    Timestamp::from_second(i64::from(ts)).map_err(D::Error::custom)
+}
```

Comment:
I'm not 100% sure, but you might be able to use `jiff::fmt::serde` here instead? https://docs.rs/jiff/latest/jiff/fmt/serde/index.html#example-timestamps-as-an-integer

Reply:
There isn't a `DateTime` here? I just mean that `jiff::fmt::serde` has helper methods for serializing and deserializing a `Timestamp` via an integer number of seconds.

Reply:
No, by `DateTime` i mean the type in ClickHouse SQL

Reply:
It looks like it's just an integer? Which is what https://docs.rs/jiff/latest/jiff/fmt/serde/timestamp/second/required/index.html will do for you.

Reply:
I think `timestamp::second` serialize/deserialize to i64 which is `DateTime64` in ClickHouse data type. Honestly, I might just change the column type to `DateTime64` one day.

Reply:
Ah I see, fair enough.